PySpice/Spice/HSpice/hspicefile/hspice_read.c

```
@@ -0,0 +1,972 @@
+		i = dateStartPosition - 1, offset = 0;
+	struct FastArray faSweep, *faPtr = NULL;
+	FILE *f = NULL;
+	PyObject *date = NULL, *title = NULL, *scale = NULL, *sweep = NULL,
+		*dataList = NULL, *sweeps = NULL,
+		*tuple = NULL, *list = NULL;
+	PyArrayObject *sweepValues = NULL, **tmpArray = NULL;
+	int *varTypes = NULL;
+	int *varSizes = NULL;
+
+	// Get hspice_read() arguments.
+	if(!PyArg_ParseTuple(args, "si", &fileName, &debugMode)) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)